src/lsm/manifest.zig

```diff
@@ -9,7 +9,7 @@ const constants = @import("../constants.zig");
 const growth_factor = constants.lsm_growth_factor;
 
 const vsr = @import("../vsr.zig");
-const table_count_max = @import("tree.zig").table_count_max;
+const table_count_max_tree = @import("tree.zig").table_count_max;
```

Comment:
There is `forest_table_count_max` below, however. This code lines up nicely as `table_count_max_tree` and the below code as `forest_table_count_max` :sweat:.